Bio/SeqUtils/CodonUsage.py

```diff
@@ -175,3 +179,199 @@ def print_index(self):
         """
         for i in sorted(self.index):
             print("%s\t%.3f" % (i, self.index[i]))
+            
+            
+class normRelativeCodonAdaptationIndex(object): 
+    """A normalized Relative Codon Adaptation index implementation. 
```

Comment:
Style wise we try to follow PEP257 for our docstrings, which means a single line summary then a blank line. The rest of the docstring indentation should match the opening quote (i.e. four spaces indented here).

This would fix the ``flake8`` style warnings like:

```
D208 Docstring is over-indented
D205 1 blank line required between summary line and description
```

See the ``CONTRIBUTING.rst`` file for more about how to run ``flake8`` locally.